simulator/tier4_dummy_object_rviz_plugin/src/tools/interactive_object.cpp

```diff
@@ -49,6 +49,7 @@
 
 #include "util.hpp"
 
+#include <QKeyEvent>
```

Comment:
Without this header, we can compile the `autoware_dummy_perception_publisher` (verified on my machine)
```suggestion
```